src/app/Instance/BridgeOverview/Components/OBEmptyState.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Button,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  Text,
+  TextContent,
+  Title,
+} from "@patternfly/react-core";
+import { PlusCircleIcon } from "@patternfly/react-icons";
+import React from "react";
+interface OBEmptyStateProps {
+  title: string;
+  description: string;
+  buttonName: string;
```

Comment:
`buttonLabel` is a better choice here, as this is the text that will be displayed to the user. 
Name could be confused with the html attribute that has a different purpose.